core/src/main/java/org/elasticsearch/common/util/concurrent/QueueResizingEsThreadPoolExecutor.java

```diff
@@ -43,17 +44,20 @@
  */
 public final class QueueResizingEsThreadPoolExecutor extends EsThreadPoolExecutor {
 
+    public static double EWMA_ALPHA = 0.3;
```

Comment:
It's totally a random pick, I'll add a comment. I'm planning on revisiting the number once all the parts are in place (so I can test different values)